magnolia-voting-app/src/main/java/com/example/votingapp/valueprovider/PollStatusValueProvider.java

```diff
@@ -0,0 +1,52 @@
+package com.example.votingapp.valueprovider;
+
+import java.util.*;
+
+import javax.jcr.*;
+
+import com.vaadin.data.*;
+
+import org.slf4j.*;
+
+import info.magnolia.jcr.*;
+
+/**
+ * This class provides value for the poll status column. 
+ */
+public class PollStatusValueProvider implements ValueProvider<Node, String> {
+    
+    private static final Logger LOGGER = LoggerFactory.getLogger(PollStatusValueProvider.class);
+
+    public static final String TO_DO = "To-do";
+    public static final String ON_GOING = "On-going";
+    public static final String DONE = "Done";
+
+    @Override
+    public String apply(Node source) {
+        return getPollStatus(source);
+    }
+
+    private static String getPollStatus(Node source) throws RuntimeException {
```

Comment:
Why do you need static method here? Do you use no-static method?

Reply:
Thank you, it was my mistake. I have changed it.